config.go

```diff
@@ -24,14 +25,21 @@ type Config struct {
 	Queues         Queues        `mapstructure:"queues" json:"queues" yaml:"queues"`
 	Producers      Producers     `mapstructure:"producers" json:"producers" yaml:"producers"`
 	Consumers      Consumers     `mapstructure:"consumers" json:"consumers" yaml:"consumers"`
+	dsnList        []string
 }
 
 // Traverses the config tree and fixes option keys name.
-func (config Config) normalize() {
+func (config *Config) normalize() {
```

Comment:
I think it makes sense to add pointers to all Config methods. What do you think?

Reply:
i think that pointers required only for methods, that may change Config object itself. like normalization does.

Reply:
Ok.